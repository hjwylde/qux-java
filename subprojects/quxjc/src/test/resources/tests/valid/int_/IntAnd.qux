package valid.int_

void main([str] args):
    print 0 and 1
    print 1 and 2 and 4
    print 0xff and 0xff
    print 1 and 1
    print 0 and 0
    print -1 and 0
    print -1 and 1
    print -1 and 2
    print 0xff and 0xffff
    print 0xffff and 0xff

