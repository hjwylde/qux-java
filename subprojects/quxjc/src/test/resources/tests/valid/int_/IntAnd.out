0
0
255
1
0
0
1
2
255
255
